### DISCUSSION TITLE
may sound odd, but use without camera?

### DISCUSSION THREAD

**Discussion by MakerBot453** [Sunoo/homebridge-camera-ffmpeg#1076] [Ideas]
This may sound a bit odd, but is it possible to use homebridge-camera-ffmpeg without a camera? Reason: I'd like to use only the 2-way audio part. Planned setup: use the computer running homebridge as a headset over bluetooth HFP for a 2-way audio device and get that audio in-out into HomeKit.

**Comment by Sunoo** [comment]
HomeKit expects some video, and will shut down the stream if it doesn’t see one. You could feed a still image or a GIF or something into the plugin though, which should satisfy that requirement.

**Reply by MakerBot453** [reply]
Thanks! How would I feed a still in the video stream??

**Reply by Sunoo** [reply]
Same way as any video file: -i followed by the path to the file

**Reply by MakerBot453** [reply]
Ok, I'll try to use a little 32x32 endless ;-).
